build/includes/terraform.mk

```diff
@@ -44,7 +44,7 @@ gcloud-terraform-cluster: $(ensure-build-image)
 gcloud-terraform-cluster: FEATURE_GATES := ""
 gcloud-terraform-cluster: GCP_PROJECT ?= $(current_project)
 gcloud-terraform-cluster:
-	$(MAKE) terraform-init DIRECTORY=gke
+	$(MAKE) terraform-init
```

Comment:
Don't we still need a `TERRAFORM_BUILD_DIR` on this line?

Reply:
terraform-init itself does not contain DIRECTORY. So it seems that this is an obsolete variable.